frameaccurate/js/frameaccurate-demo.js

```diff
@@ -116,15 +132,35 @@ function step(stepSize) {
     let errorField = document.querySelector('.smpte-error');
     errorField.style.display = 'none';
 
+    // allow numeric values
+    const targetSmpte = convertNumericSMPTE(smtpeSeekTime.value);
+
     try {
-      smtpeController.seekToSMPTE(smtpeSeekTime.value);
+      smtpeController.seekToSMPTE(targetSmpte);
     }
     catch(err) {
       errorField.innerHTML = err;
       errorField.style.display = 'block';
     }
   }
 
+  function convertNumericSMPTE(smpteValue) {
+    if (isFinite(smpteValue)) {
+      const padNum = function(num){
```

Comment:
Why did you embed the `padNum` function declaration within the `convertNumericSMPTE` function?